Runtime/AvatarCreator/WebRequests/AuthorizedRequest.cs

```diff
@@ -23,15 +21,8 @@ public async Task<T> SendRequest<T>(RequestData requestData, CancellationToken c
 
             if (response is { IsSuccess: false, ResponseCode: 401 })
             {
-                try
-                {
-                    await AuthManager.RefreshToken();
-                }
-                catch (Exception)
-                {
-                    throw;
-                }
-                
+                await AuthManager.RefreshToken();
+
```

Comment:
I would suggest not removing this, if I remember correctly that this was added for when even RefreshToken request fails and then without it the error is not caught and propagated above. But somehow rider always suggest to remove it.

Reply:
Oh, interesting, it seemed to be redundant but if it's a weird edge case then I'll add it back 👍

Reply:
Added back, and also added a comment on it too so others don't make the same assumption in the future that I did.